sentry/src/routes/channel.rs

```diff
@@ -596,4 +629,105 @@ mod test {
             assert_eq!(expected, res.expect_err("Should return an error"));
         }
     }
+
+    #[tokio::test]
+    async fn adds_and_retrieves_spender_leaf() {
+        let app = setup_dummy_app().await;
+        let channel = DUMMY_CAMPAIGN.channel;
+
+        insert_channel(&app.pool, channel)
+            .await
+            .expect("should insert channel");
+
+        let build_request = |channel: Channel| {
+            Request::builder()
+                .extension(channel)
+                .body(Body::empty())
+                .expect("Should build Request")
+        };
+        let build_request_with_params = |channel: Channel| {
+            let param = RouteParams(vec![
+                channel.id().to_string(),
+                ADDRESSES["creator"].to_string(),
```

Comment:
Refrain from using `ADDRESSES` and instead use the static in the module:
`primitives::test_util::{ADVERTISER, CREATOR, PUBLISHER, PUBLISHER_2, ...}`